File: SproInstall/SocketPro/samples/RAdo/VBNet/Client/RAdo_i.vb
Public Class RAdoConst
	'defines for service CRAdo
    Public Const sidCRAdo As Integer = (CInt(USOCKETLib.tagOtherDefine.odUserServiceIDMin) + 202)

    Public Const idGetDataSetCRAdo As Short = (CShort(USOCKETLib.tagOtherDefine.odUserRequestIDMin) + 0)
	Public Const idGetDataReaderCRAdo As Short = (idGetDataSetCRAdo + 1)
	Public Const idSendDataSetCRAdo As Short = (idGetDataReaderCRAdo + 1)
	Public Const idSendDataReaderCRAdo As Short = (idSendDataSetCRAdo + 1)
End Class

File: SproInstall/SocketPro/tutorial/CSharp/SampleFour/Client/RAdo_i.cs
public class RAdoConst
{
	//defines for service CRAdo
	public const int sidCRAdo = ((int)USOCKETLib.tagOtherDefine.odUserServiceIDMin + 202);

	public const short idGetDataSetCRAdo = ((short)USOCKETLib.tagOtherDefine.odUserRequestIDMin + 0);
	public const short idGetDataReaderCRAdo = (idGetDataSetCRAdo + 1);
	public const short idSendDataSetCRAdo = (idGetDataReaderCRAdo + 1);
	public const short idSendDataReaderCRAdo = (idSendDataSetCRAdo + 1);
}